packages/forms/src/Concerns/HasStateBindingModifiers.php

```diff
@@ -42,6 +42,10 @@ public function applyStateBindingModifiers(string $expression): string
     {
         $modifiers = $this->getStateBindingModifiers();
 
+        if (str_starts_with($expression, 'entangle') && (in_array('lazy', $modifiers) || in_array('debounce', $modifiers))) {
+            $modifiers = ['defer'];
```

Comment:
I think I disagree, entangling lazy and debounce fields should mean that no modifiers are applied, as people are still expecting those fields to refresh the Livewire component when they are changed

Reply:
I've fixed this